certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureDigitalTwins",
+    "description": "Azure Digital Twins",
+    "version": "1.0"
+  },
+  "host": "digitaltwins.azure.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/digitaltwins/{twinid}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "type": "string",
+                  "description": "Results From the Twin.",
+                  "title": "Result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Twin by Id",
+        "description": "Pass in the twin id to retrieve its details.",
+        "operationId": "GetTwinById",
+        "parameters": [
+          {
+            "name": "twinid",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2020-10-31",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      },
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@sankumrav empty response schemas are not allowed and using only default responses is not allowed.

Reply:
Azure digital twins have dynamic schemas and the response depends on the customer's model. It's not possible to define a schema upfront.

Reply:
https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-2-meet-submission-requirements 

There is a point where it allows exceptions: "Empty response schemas aren't allowed except in special cases where the response schema is dynamic. This means no dynamic content will be shown in the output and makers must use JSON to parse the response."

Reply:
Nevermind. Fixed it.